cmd/manager/main.go

```diff
@@ -97,6 +97,8 @@ func main() {
 	// Set default manager options
 	options := manager.Options{
 		Namespace: namespace,
+		// Disable controller-runtime metrics serving
+		MetricsBindAddress: "0",
```

Comment:
It didn't look like it from the output you pasted, but we won't be losing anything by disabling this?

Reply:
This is actually fixing a bug where our current custom metrics are being overridden by the controller-runtime metrics server. If you look at certman-operator on hive-stage there are no custom metrics being served. So in fact we are gaining, not disabling things here. 👍

Reply:
:+1: